Formula/glooctl.rb

```diff
@@ -2,15 +2,15 @@ class Glooctl < Formula
   desc "Envoy-Powered API Gateway"
   homepage "https://gloo.solo.io"
   url "https://github.com/solo-io/gloo.git",
-      :tag      => "v0.19.1",
-      :revision => "1539067a393f53a0b793106aaa4676f8ff87df99"
+      :tag      => "v0.20.1",
```

Comment:
This is not based on the top of `master` (which already has 0.20.1)

Reply:
Hmm I'll update our fork and then update this PR. This PR would merge into current master as expected though, it's just the diff is a little confusing

Reply:
updated! https://github.com/Homebrew/homebrew-core/pull/44773/commits/b00d86b41b38842c8e692ed37b6d826a0df04ce3

Reply:
> This PR would merge into current master as expected though

The github UI can do it, but we merge formulas through a command-line (because of the binary bottles) which requires things to apply on top of master.